### PR TITLE
build(storage-resize-images) replace `mkdirp-promise` with `mkdirp`

### PR DESCRIPTION
**Pull request by sudokzt**
Use mkdirp instead of mkdirp-promise Because this package has been deprecated.
[NPM](https://www.npmjs.com/package/mkdirp-promise)

### CODE REVIEW

**Comment by sudokzt** [comment]
@googlebot I signed it!

**Comment by sudokzt** [comment]
@russellwheatley 
Thank you for reviewing🙌 I did remove `yarn.lock`, then run `yarn`.

**Comment by sudokzt** [comment]
@googlebot I fixed it.

**Comment by sudokzt** [comment]
@googlebot I fixed it.

**Comment by sudokzt** [comment]
@googlebot I fixed it.

**Comment by sudokzt** [comment]
@googlebot I fixed it.

**Comment by sudokzt** [comment]
@googlebot I fixed it.

**Comment by sudokzt** [comment]
@googlebot I signed it!

**Comment by sudokzt** [comment]
@russellwheatley 
Thank you for reviewing. I'm sorry, I wasn't careful.
I removed `yarn.lock`,  then run `npm install`.

File: storage-resize-images/package.json [hunk 1]
```diff
@@ -14,9 +14,8 @@
   "dependencies": {
     "firebase-admin": "^8.0.0",
     "firebase-functions": "^3.3.0",
-    "mkdirp": "^0.5.1",
-    "mkdirp-promise": "^4.0.0",
-    "sharp": "0.23.4"
+    "mkdirp": "^1.0.4",
+    "sharp": "^0.24.1"
```
Comment (russellwheatley) [inline storage-resize-images/package.json:18]: Sorry, I didn't spot this earlier. The `sharp` library needs fixing to this version `"sharp": "0.23.4"` to ensure compatibility with Node version 8. See this [PR](https://github.com/firebase/extensions/pull/278).

**Comment by sudokzt** [comment]
@russellwheatley 
Thank you. I fixed it.

**Comment by sudokzt** [comment]
@googlebot  I fixed it.

**Comment by sudokzt** [comment]
@googlebot I fixed id.

**Comment by sudokzt** [comment]
@russellwheatley 
Thank you for everything. Should I wait for the merge?

**Comment by russellwheatley** [comment]
Hey @sudokzt, I think the problem is you may have branched off from `master` instead of `next` branch as there are a bunch of commits that shouldn't be on the PR which I think are causing the checks to fail. I'm afraid you may have to create a fresh PR for these changes to be merged.

**Pull request merged**